docs/market_analysis.rst

```diff
@@ -0,0 +1,291 @@
+So for example:
+
+===== ===== ==== =========
+Time  Short Long Suggested
+===== ===== ==== =========
+12:00 0.08  0.1  0.1
+12:01 0.09  0.1  0.1
+12:02 0.1   0.1  0.105
+12:03 0.11  0.1  0.1155
+12:04 0.12  0.1  0.126
+===== ===== ==== =========
+
+In this example, the bot would start to lend at 12:02 and it would suggest a minimum lending rate of 0.1 * `MACD_multiplier`_, which by default is 1.05. Giving a rate of 0.105. This is then passed back to the main lendingbot where it will use your gaptop and gapbottom, along with spreads and all the other smarts to place loan offers.
+
+Currently using this method gives the best results with well configured gaptop and gapbottom. This allows you to catch spikes in the market as see above. 
+
+The short window and long window are configured by a number of seconds, the data is then taken from the DB requesting `MACD_long_win_seconds`_ * 1.1. This is to get an extra 10% of data as there is usually some lost in the recording from Poloniex.
+You can also use the `data_tolerance`_ to help with the amount of data required by the bot for this calculation, that is the number of seconds that can be missing for the data to still be valid.
+
+This current implementation is basic in it's approach, but will be built upon with time. Results seem to be good though and we would welcome your feedback if you play around with it.
+
+suggested loan rate
+'''''''''''''''''''
+If the average of the short window is greater than the average of the long window we will return the current
+
+configuring
+'''''''''''
+
+The number of config options and combinations for this can be quite daunting. As time goes on I hope more people will feed back useful figures for all our different configuration set ups. I have put in sensible defaults into the config for the MACD section. These are options that I have changed that aren't set by default and work better if you're using MACD as the rate calculation method. Change the currency to whatever you want, though best not use more than 3 really, as it slows down the calls to poloniex considerably. If you can use just one, then do it.
+
+I'm hoping that once more people test and report back results, this can be updated and more information passed to everyone else. 
+
+The most important is probably the hidecoins change to False. This means that it will always place loans so you don't need to have as low a resolution on the sleep timers. You also want to make sure the gaptop and gapbottom are high so you can get a large spread.
+
+======================= =========
+Config                  Value
+======================= =========
+sleeptimeactive         10
+sleeptimeinactive       10
+spreadlend              3
+gapbottom               400
```

Comment:
I'm running some tests now to get a better value for this, then I'll move it to RawBTC.